template-tags.php

```diff
@@ -448,6 +448,7 @@ function coauthors_wp_list_authors( $args = array() ) {
 		'style'            => 'list',
 		'html'             => true,
 		'number'           => 20, // A sane limit to start to avoid breaking all the things
+		'guest_authors_only' => false
```

Comment:
We should keep the spacing consistent here, and the doc block will need updating to include this new argument. (The formatting of the doc block is also not following the standards, so we can sort that at the same time.